# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/mine/gasgui-devs.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/gasgui.c.vanilla	2021-05-07 23:29:11.877425374 +0200
+++ mine-0.23/gasgui.c	2021-05-07 23:30:38.061421938 +0200
@@ -34,9 +34,9 @@
 #endif
 
 char * config    = ROCKCFG_ID;
-char * targetdir = "/mnt/target";
-char * sourcedev = "/dev/cdroms/cdrom0";
-char * sourcedir = "/mnt/cdrom0";
+char * targetdir = "/mnt";
+char * sourcedev = "/dev/sr0";
+char * sourcedir = "/media/cdrom";
 char * sourceopt = "ro";
 
 char * mine_command = "mine";
@@ -56,10 +56,10 @@
 "  -t /mnt/target\n"
 "		The root directory of the target system.\n"
 "\n"
-"  -d /dev/cdroms/cdrom0\n"
+"  -d /dev/sr0\n"
 "		The source device file.\n"
 "\n"
-"  -s /mnt/cdrom0\n"
+"  -s /media/cdrom\n"
 "		The mount point for the source device file (or URL).\n"
 "\n"
 "  -o ro\n"
